### PR TITLE
- added task 1

### PR DESCRIPTION
**Pull request by 1994arsensargsyan1994**

### CODE REVIEW

File: arsen-sargsyan-resources/readFile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+arsen hello barev nidodon hello
```
Comment (serghovhannisyan) [inline arsen-sargsyan-resources/readFile]: maybe there is a way to put this file inside your package?

File: src/com/egs/arsensargsyan/task1/ReadHello.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ReadHello {
+
+    private static final String PATH_FILE = "arsen-sargsyan-resources/readFile";
+    private static final String RESULT_FILE = "arsen-sargsyan-resources/result";
+
+    private Map<Integer, Integer> resultMap = new HashMap<>();
```
Comment (M-Suzy) [inline src/com/egs/arsensargsyan/task1/ReadHello.java:12]: I think you don't need a Map for this problem.

File: src/com/egs/arsensargsyan/task1/ReadHello.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    public static void main(String[] args) {
+        new ReadHello().go();
+    }
+
+    public void go() {
+        try {
+            resultMap = read("hello");
```
Comment (M-Suzy) [inline src/com/egs/arsensargsyan/task1/ReadHello.java]: Keep hello as const

**Review by serghovhannisyan — APPROVED** [review]

**Review by serghovhannisyan — CHANGES_REQUESTED** [review]
arsen-sargsyan-resources/ folder still there

**Review by serghovhannisyan — APPROVED** [review]

**Pull request merged**